### PR TITLE
Add-track flyout simplification

### LINKED ISSUE #7354: [AU4] Tracks - Add new track v2
### Description

Simplified implementation of the add new track fly-out.

[Figma link](https://www.figma.com/design/FWiArQcqiLRB4a01w9s1lV/Tracks?node-id=4312-13065)

### PR DESCRIPTION
**Pull request by saintmatthieu**
Resolves: #7354

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

File: au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/AddNewTrackPopup.qml [hunk 1]
```diff
@@ -18,170 +18,49 @@ StyledPopupView {
...
+                model: [
+                    { type: TrackType.MONO, icon: IconCode.MICROPHONE, text: qsTrc("projectscene", "Mono") },
+                    { type: TrackType.STEREO, icon: IconCode.MICROPHONE, text: qsTrc("projectscene", "Stereo") },
+                    { type: TrackType.LABEL, icon: IconCode.LOOP_IN, text: qsTrc("projectscene", "Label") }
+                ]
+
+                FlatButton {
+                    Layout.preferredWidth: 64
+                    Layout.preferredHeight: 64
+                    Layout.fillHeight: true
```
Comment (Eism) [inline au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/AddNewTrackPopup.qml:44]: please add empty lines between property groups, it helps in code readability
see https://github.com/musescore/MuseScore/wiki/CodeGuidelines#qml

File: au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/AddNewTrackPopup.qml [hunk 2]
```diff
@@ -18,170 +18,49 @@ StyledPopupView {
...
+
+            Repeater {
+                model: [
+                    { type: TrackType.MONO, icon: IconCode.MICROPHONE, text: qsTrc("projectscene", "Mono") },
+                    { type: TrackType.STEREO, icon: IconCode.MICROPHONE, text: qsTrc("projectscene", "Stereo") },
+                    { type: TrackType.LABEL, icon: IconCode.LOOP_IN, text: qsTrc("projectscene", "Label") }
+                ]
+
+                FlatButton {
+                    Layout.preferredWidth: 64
```
Comment (grliszas14) [inline au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/AddNewTrackPopup.qml:42]: according to the design, buttons should be 80px wide and 72px tall

File: au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/AddNewTrackPopup.qml [hunk 3]
```diff
@@ -18,170 +18,49 @@ StyledPopupView {
...
-                    } else if (value < numberOfTracksValidator.top) {
-                        prv.numberOfTracks = value + 1
-                    }
-
-                    input.focus = false
                 }
             }
         }
 
         SeparatorLine {}
```
Comment (grliszas14) [inline au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/AddNewTrackPopup.qml:56]: don't know how important is that but on the design SeparatorLine stretches exactly from left flyout edge to the right (right now it has a few pixels margin)

Comment (saintmatthieu) [reply au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/AddNewTrackPopup.qml]: The design is the design. I'll address.

**Review by grliszas14 — APPROVED** [review]

File: au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/AddNewTrackPopup.qml (hunk 1 above)
Comment (saintmatthieu) [reply au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/AddNewTrackPopup.qml]: Will address after rebase and before merge, will save a build run.

**Pull request merged**